### PR TITLE
feat: add convnext model and config

### PR DESCRIPTION
**Pull request by qizhaoaoe**
1. add convnext model
2. add config file(fp32)

### CODE REVIEW

File: plsc/models/convnext.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    with shape (batch_size, channels, height, width).
+    """
+
+    def __init__(self, normalized_shape, eps=1e-6,
+                 data_format="channels_last"):
+        super().__init__()
+        # print("normalized_shape: ", normalized_shape)
+        self.weight = paddle.create_parameter(
+            shape=(normalized_shape, ),
+            dtype="float32",
```
Comment (GuoxiaWang) [inline plsc/models/convnext.py:45]: 创建参数的时候为什么需要加 dtype 呢？

File: plsc/models/convnext.py [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        self.pwconv1 = nn.Linear(dim, 4 * dim)
+        self.act = nn.GELU()
+        self.pwconv2 = nn.Linear(4 * dim, dim)
+        self.gamma = None
+        if layer_scale_init_value > 0:
+            self.gamma = paddle.create_parameter(
+                shape=(dim, ),
+                dtype="float32",
+                default_initializer=paddle.nn.initializer.Assign(
+                    paddle.ones((dim, )) * layer_scale_init_value))
```
Comment (GuoxiaWang) [inline plsc/models/convnext.py:91]: `paddle.ones((dim, )) * layer_scale_init_value` 可以用 `paddle.fill` 替换

File: plsc/models/convnext.py [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+            self.set_dict(state_dict)
+
+    def save(self, path, local_rank=0, rank=0):
+        if local_rank == 0:
+            paddle.save(self.state_dict(), path + ".pdparams")
+
+
+def convnext_tiny(pretrained=False, in_22k=False, checkpoint=None, **kwargs):
+    model = ConvNeXt(depths=[3, 3, 9, 3], dims=[96, 192, 384, 768], **kwargs)
+    if pretrained:
```
Comment (GuoxiaWang) [inline plsc/models/convnext.py:212]: 权重加载这里对照一下其他模型实现

Comment (qizhaoaoe) [reply plsc/models/convnext.py]: done

File: plsc/models/convnext.py (hunk 2 above)
Comment (qizhaoaoe) [reply plsc/models/convnext.py]: replace it by paddle.nn.initializer.Constant(value).

File: plsc/models/convnext.py (hunk 1 above)
Comment (qizhaoaoe) [reply plsc/models/convnext.py]: replace it by self.create_parameter().
